lib/Sema/CSWalker.cpp

```diff
@@ -205,6 +205,67 @@ class LocalCSWalker : public glu::ast::ASTWalker<LocalCSWalker, void> {
             )
         );
     }
+
+    void postVisitUnaryOpExpr(glu::ast::UnaryOpExpr *node)
```

Comment:
this is the same as a call, `+a` -> `` `+`(a)``, so no need to duplicate code. Difference is that we don't have to check for variables here, just functions.